python/arcticdb/version_store/library.py

```diff
@@ -1253,6 +1253,7 @@ def snapshot(
         metadata: Any = None,
         skip_symbols: Optional[List[str]] = None,
         versions: Optional[Dict[str, int]] = None,
+        allow_partial_snapshot: Optional[bool] = False,
```

Comment:
Let's leave this off the `Library` API till there is demand for it? Once it's on here we will never be able to get rid of it